docs/src/main/asciidoc/secretmanager.adoc

```diff
@@ -101,6 +101,7 @@ The `SecretManagerTemplate` class simplifies operations of creating, updating, a
 
 To begin using this class, you may inject an instance of the class using `@Autowired` after adding the starter dependency to your project.
 
+Note that you should add `spring.config.import=sm://` in your `application.properties` file to enable `SecretManagerTemplate` bean creation.
```

Comment:
Should we also add it to the migration guide?
Alternatively, can this be done automatically in the auto-configuration?